packages/core/src/provider/base.ts

```diff
@@ -396,15 +396,19 @@ export class SmartAccountProvider<
     const { maxFeePerGas, maxPriorityFeePerGas } =
       await this._runMiddlewareStack(uoToSubmit, overrides);
 
+    const _maxPriorityFeePerGas = bigIntMax(
+      BigInt(maxPriorityFeePerGas ?? 0n),
+      bigIntPercent(uoToDrop.maxPriorityFeePerGas, 110n)
+    );
     const _overrides: UserOperationOverrides = {
       maxFeePerGas: bigIntMax(
         BigInt(maxFeePerGas ?? 0n),
-        bigIntPercent(uoToDrop.maxFeePerGas, 110n)
-      ),
-      maxPriorityFeePerGas: bigIntMax(
-        BigInt(maxPriorityFeePerGas ?? 0n),
-        bigIntPercent(uoToDrop.maxPriorityFeePerGas, 110n)
+        bigIntPercent(
+          BigInt(uoToDrop.maxFeePerGas) - _maxPriorityFeePerGas,
```

Comment:
I think we need this:
```suggestion
          BigInt(uoToDrop.maxFeePerGas) - uoToDrop.maxPriorityFeePerGas,
```

That way, we correctly get:
```
const _maxPriorityFeePerGas = max (1.1 * uoToDrop.maxPriorityFeePerGas, maxPriorityFeePerGas)

const new_baseFeePerGas = 1.1 * (old_maxFeePerGas - old_maxPriorityFeePerGas)

const new_maxFeePerGas = new_baseFeePerGas + _maxPriorityFeePerGas
```

Reply:
thanks @avasisht23